conn/raft_server.go

```diff
@@ -120,20 +121,22 @@ func (p *proposals) Done(key string, err error) {
 	pd.Ch <- err
 }
 
-func (w *RaftServer) GetNode() *Node {
+func (w *RaftServer) getNode() *Node {
 	w.nodeLock.RLock()
 	defer w.nodeLock.RUnlock()
 	return w.Node
```

Comment:
This is not protecting the Node from being modified by different goroutines at a time.

Reply:
True. I'll look further into this. For now, I've added a TODO to look into this.